tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "system-test/**/*.ts"
   ],
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "test/fixtures"
```

Comment:
Is this line necessary?

Reply:
Not anymore.  I originally had test code in the fixtures directory, but its not there now.